smack-core/src/main/java/org/jivesoftware/smack/ReconnectionManager.java

```diff
@@ -26,19 +26,25 @@
 import java.util.WeakHashMap;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+
 /**
  * Handles the automatic reconnection process. Every time a connection is dropped without
  * the application explicitly closing it, the manager automatically tries to reconnect to
  * the server.<p>
+ * 
+ * There are two possible reconnection policies:
  *
- * The reconnection mechanism will try to reconnect periodically:
+ * {@link ReconnectionPolicy#RANDOM_INCREASING_DELAY} - The reconnection mechanism will try to reconnect periodically:
  * <ol>
  *  <li>For the first minute it will attempt to connect once every ten seconds.
  *  <li>For the next five minutes it will attempt to connect once a minute.
  *  <li>If that fails it will indefinitely try to connect once every five minutes.
  * </ol>
+ * 
+ * {@link ReconnectionPolicy#FIXED_DELAY} - The reconnection mechanism will try to reconnect after a fixed delay 
+ * independently from the number of reconnection attempts already performet
  *
- * @author Francisco Vives
+ * @author Francisco Vives, Luca Stucchi
```

Comment:
since you are at it, please add an extra author tag for you, not touching the existing one